backend/wmg/data/load_corpus.py

```diff
@@ -64,6 +64,15 @@ def load_h5ad(h5ad: str, group_name: str, validate: bool):
     if not validate_dataset_properties(anndata_object):
         return
 
+    # Set all values <=min_value to 0
+    min_value = 3
```

Comment:
To be moved to a constant together with the other optimization (the 500 filter) - I'll consolidate them on the other PR.